ui/app/mainui/SendModalHandler.qml

```diff
@@ -308,7 +313,8 @@ QtObject {
         let params = {}
         if (root.simpleSendEnabled) {
             params = {
-                selectedRecipientAddress: recipientAddress
+                selectedRecipientAddress: recipientAddress,
+                openReason: "send to recipient"
```

Comment:
Hey @Cuteivist not sure why you need this openReason here, the sendType is being set for all of them and can be used later in the function you created `getSendTypeString` to decide open reason? wdyt ?